module/Makefile.in

```diff
@@ -134,17 +134,19 @@ cppcheck-Linux:
 		-I @top_srcdir@/include/os/linux/spl \
 		-I @top_srcdir@/include/os/linux/zfs \
 		-I @top_srcdir@/include \
-		avl icp lua nvpair spl unicode zcommon zfs zstd os/linux
+		avl icp lua nvpair unicode zcommon zfs zstd os/linux
 
 cppcheck-FreeBSD:
 	@true
 
 cppcheck: cppcheck-@ac_system@
 
 distdir:
-	(cd @srcdir@ && find $(ZFS_MODULES) os -name '*.[chS]') | \
-	while read path; do \
-		mkdir -p $$distdir/$${path%/*}; \
-		cp @srcdir@/$$path $$distdir/$$path; \
-	done; \
+	cd @srcdir@ && find . -name '*.[chS]' -exec sh -c 'for f; do mkdir -p $$distdir/$${f%/*}; cp @srcdir@/$$f $$distdir/$$f; done' _ {} +
 	cp @srcdir@/Makefile.bsd $$distdir/Makefile.bsd
+
+gen-zstd-symbols:
+	for obj in $(addprefix zstd/,$(ZSTD_UPSTREAM_OBJS)); do echo; echo "/* $${obj#zstd/}: */"; @OBJDUMP@ -t $$obj | awk '$$2 == "g" && !/ zfs_/ {print "#define\t" $$6 " zfs_" $$6}' | sort; done >> zstd/include/zstd_compat_wrapper.h
+
+check-zstd-symbols:
+	@OBJDUMP@ -t $(addprefix zstd/,$(ZSTD_UPSTREAM_OBJS)) | awk '/file format/ {print}  $$2 == "g" && !/ zfs_/ {++ret; print}  END {exit ret}'
```

Comment:
This code is pretty dense.  Would you mind adding a line or two describing what it's doing?

Reply:
I mean, it's the same code that lived in `module/zstd/Makefile.in`, and it's as straightforward then as it is now (the move meant `$(addprefix` and `#zstd/}`) and explained in zstd_compat_wrapper.h:
```c
/*
 * This wrapper fixes a problem, in case the ZFS filesystem driver, is compiled
 * statically into the kernel.
 * This will cause a symbol collision with the older in-kernel zstd library.
 *
 * On update, truncate this file at the scissor line, rebuild the module,
 * and make gensymbols.
 */

...

/* -- >8 -- */

/* lib/common/entropy_common.o: */
#define	FSE_getErrorName zfs_FSE_getErrorName
#define	FSE_isError zfs_FSE_isError
#define	FSE_readNCount zfs_FSE_readNCount
#define	FSE_versionNumber zfs_FSE_versionNumber
...
```

Reply:
Ah nevemind, I see the old code now (`gensymbols`/`checksymbols`).  You can leave it as-is.